mora/util.py

```diff
@@ -97,10 +99,16 @@ def parsedatetime(s: str) -> datetime.datetime:
     try:
         dt = dateutil.parser.parse(s, dayfirst=True, tzinfos=_tzinfos)
     except ValueError:
-        raise exceptions.HTTPException(
-            exceptions.ErrorCodes.E_INVALID_INPUT,
-            'cannot parse {!r}'.format(s)
-        )
+        if default is not _sentinel:
```

Comment:
Do we actually need a sentinel here? Is having `default` be `None` in the signature, and having `if default is not None` not enough?